tests/src/Network/ProbeTest.php

```diff
@@ -105,4 +116,122 @@ public function testGetFeedLinkBase()
+				'assertUri'   => 'Artists4Future_Muenchen@climatejustice.global',
+				'assertInfos' => [
+					'name'         => 'Artists4Future München',
+					'nick'         => 'Artists4Future_Muenchen',
+					'url'          => 'https://climatejustice.global/users/Artists4Future_Muenchen',
+					'alias'        => 'https://climatejustice.global/@Artists4Future_Muenchen',
+					'photo'        => 'https://cdn.masto.host/climatejusticeglobal/accounts/avatars/000/021/220/original/05ee9e827a5b47fc.jpg',
+					'header'       => 'https://cdn.masto.host/climatejusticeglobal/accounts/headers/000/021/220/original/9b98b75cf696cd11.jpg',
+					'account-type' => 0,
+					'about'        => 'Wir sind Künstler oder einfach gerne kreativ tätig und setzen uns unabhängig von politischen Parteien für den Klimaschutz ein. Die Bedingungen zu schaffen, die die [url=https://climatejustice.global/tags/Klimakrise]#Klimakrise[/url] verhindern/eindämmen (gemäß den Forderungen der [url=https://climatejustice.global/tags/Fridays4Future]#Fridays4Future[/url]) ist Aufgabe der Politik, muss aber gesamtgesellschaftlich getragen werden. Mit unseren künstlerischen Aktionen wollen wir einen anderen Zugang anbieten für wissenschaftlich rationale Argumente, speziell zur Erderwärmung und ihre Konsequenzen.',
+					'hide'         => 0,
+					'batch'        => 'https://climatejustice.global/inbox',
+					'notify'       => 'https://climatejustice.global/users/Artists4Future_Muenchen/inbox',
+					'poll'         => 'https://climatejustice.global/users/Artists4Future_Muenchen/outbox',
+					'subscribe'    => 'https://climatejustice.global/authorize_interaction?uri={uri}',
+					'following'    => 'https://climatejustice.global/users/Artists4Future_Muenchen/following',
+					'followers'    => 'https://climatejustice.global/users/Artists4Future_Muenchen/followers',
+					'inbox'        => 'https://climatejustice.global/users/Artists4Future_Muenchen/inbox',
+					'outbox'       => 'https://climatejustice.global/users/Artists4Future_Muenchen/outbox',
+					'sharedinbox'  => 'https://climatejustice.global/inbox',
+					'priority'     => 0,
+					'network'      => 'apub',
+					'pubkey'       => '-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEA6pYKPuDKb+rmBB869uPV
+uLYFPosGxMUfenWqfWmFKzEqJ87rAft0IQDAL6dCoYE55ov/lEDNROhasTZLirZf
+M5b7/1JmwMrAfEiaciuYqDWT3/yDpnekOIdzP5iSClg4zt7e6HRFuClqo4+b6hIE
+DTMV4ksItvq/92MIu62pZ2SZr5ADPPZ/914lJ86hIH5BanbE8ZFzDS9vJA7V74rt
+Vvkr5c/OiUyuODNYApSl87Ez8cuj8Edt89YWkDCajQn3EkmXGeJY/VRjEDfcyk6r
+AvdUa0ArjXud3y3NkakVFZ0d7tmB20Vn9s/CfYHU8FXzbI1kFkov2BX899VVP5Ay
+xQIDAQAB
+-----END PUBLIC KEY-----',
+					'manually-approve' => 0,
+					'baseurl'          => 'https://climatejustice.global',
+				]
+			]
+		];
+	}
+
+	/**
+	 * @dataProvider dataUri
+	 */
+	public function testCleanUri(string $uri, string $assertUri, array $assertInfos)
+	{
+		self::markTestIncomplete('hard work due mocking 19 different http-requests');
```

Comment:
`Network\Probe::cleanUri()` doesn't perform any remote HTTP calls, are you confusing with `Network\Probe::uri()`?

Reply:
yes indeed, I merged both tests :-)